Refresh.GameServer/Authentication/GameAuthenticationProvider.cs

```diff
@@ -31,6 +32,17 @@ public GameAuthenticationProvider(GameServerConfig? config)
             tokenData = request.RequestHeaders["Authorization"];
         }
 
+        // ReSharper disable once SwitchExpressionHandlesSomeKnownEnumValuesWithExceptionInDefault
+        string validBaseRoute = tokenType switch
+        {
+            TokenType.Game => GameEndpointAttribute.BaseRoute,
+            TokenType.Api => ApiV3EndpointAttribute.BaseRoute,
```

Comment:
Do any api v1 endpoints require auth? if so this will break in that case

Reply:
No, our implementation of Lighthouse's API doesn't include any authentication endpoints.

Most of their stuff is done through their Razor front-end which doesn't apply to us.

Reply:
Should be noted that Lighthouse *does* have authenticated API endpoints, but they're only for sending out invite codes automatically for beta testing. I don't think it would be too difficult to implement in `LegacyApi`, but that one feature is irrelevant to us since we never had that stage so it hasn't been implemented. Another thing is that the token for that feature isn't attached to a user from what I remember correctly so that type of token wouldn't pass through our authentication provider anyways

I wouldn't mind adding authentication support for `LegacyApi` if they ever expand their API, but I don't know if they've thought about doing that in recent times. I think they just don't have a reason to.